plugins/node/opentelemetry-instrumentation-graphql/test/graphql.test.ts

```diff
@@ -572,6 +572,99 @@ describe('graphql', () => {
     });
   });
 
+  describe('when ignoreFields has a property', () => {
```

Comment:
Currently, instrumentation tests only with resolvers defined on the schema itself, but there are no tests for the patching of the "fieldResolver" (or defaultFieldResolver).

This PR adds logic that is currently not tested for the "fieldResolver".
Could be amazing to test your logic for this flow as well. Since there are no tests for this even for other features, I am ok if you choose not to do it.